www/_template/guides.md

```diff
@@ -4,74 +4,6 @@ title: Guides
 description: Snowpack's usage and integration guides.
 ---
 
-<style>
```

Comment:
I appreciate trying to bring these together, but the two had different CSS properties (`/news` was a 3-across grid, `/guides` was a 4-across grid.

Could you rename `.news-items` into a generic `.card-grid` CSS class, and then set the sizing via some other class? Something like:

```
.card-grid {
    display: grid;	
    grid-column-gap: 15px;	
    grid-row-gap: 15px;	
  grid-auto-flow: dense;
}
.card-grid.is-3 {	
    grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
}
.card-grid.is-4 {
    grid-template-columns: repeat(auto-fit, minmax(200px, 1fr));	
}
```